docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -471,7 +470,46 @@ To encode a raw image:
    ```
 
 <a name="animation"></a>
-## Encoding an Animated GIF
+## Encoding an Animated GIF or WEBP
```

Comment:
Global: Please use sentence case for all headings

Reply:
ok! thank you for the comment.